docs/site/tutorials/protocol_oriented_generics.ipynb

```diff
@@ -0,0 +1,1108 @@
+{
```

Comment:
<p>(regarding `FilterPredicate`)</p><p><strong>Eugene:</strong></p><blockquote>Why not use a function type instead?</blockquote><p><strong>Bart:</strong></p><blockquote>That's definitely another option! I thought of using another protocol, but it can be a bit overkill here.&nbsp;</blockquote><blockquote><br></blockquote><blockquote>Let me know what you think, as to whether just have a function, keep it with the protocol, or show both examples.</blockquote>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/swift/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
@burmako Wanted to see what you think